apps/frontend/src/hooks/search.ts

```diff
@@ -40,7 +41,7 @@ export const useSearch = () => {
     searchQuery,
     debouncedSearchQuery,
     setSearchQuery,
-    filters,
+    filters: new Set<string>(currentFilters),
```

Comment:
We should probably add a unit test for this. While I know what this does now there is no guarantee someone would in the future without a test